src/components/monitor-list/monitor-list.jsx

```diff
@@ -2,31 +2,31 @@ const React = require('react');
 const Box = require('../box/box.jsx');
 const Monitor = require('../../containers/monitor.jsx');
 const PropTypes = require('prop-types');
+const {OrderedMap} = require('immutable');
+
 
 const styles = require('./monitor-list.css');
 
 const MonitorList = props => (
     <Box
         className={styles.monitorList}
     >
-        {props.monitors.map(monitorData => (
+        {props.monitors.valueSeq().map(monitorData => (
             <Monitor
-                {...monitorData}
+                color={monitorData.color}
                 key={monitorData.id}
+                label={monitorData.label}
+                value={monitorData.value}
+                x={monitorData.x}
+                y={monitorData.y}
```

Comment:
https://github.com/facebook/immutable-js/issues/936#issuecomment-236502840

Found this discussion about it. We decided offline that this is better than converting to an object and then destructuring.